samples/test/cucumber.wdio.conf.ts

```diff
@@ -29,7 +29,7 @@ if (CI) {
 
 // driver version must match installed chrome version
 // https://chromedriver.storage.googleapis.com/index.html
-const CHROMEDRIVER_VERSION = '89.0.4389.23';
+const CHROMEDRIVER_VERSION = '91.0.4472.19';
```

Comment:
The default here should match the version on Bacon (which is 89).  How about allowing it to be set via env var:

```
const CHROMEDRIVER_VERSION = '89.0.4389.23' || process.env.CHROMEDRIVER_VERSION

```

Then it can be put in the `testenv.yml` file

Reply:
probably should be `const CHROMEDRIVER_VERSION = process.env.CHROMEDRIVER_VERSION || '89.0.4389.23'`

Reply:
yes, thank you.